ui/job-view/JobView.jsx

```diff
@@ -92,12 +139,21 @@ class JobView extends React.Component {
     });
   }
 
+  checkServerRevision() {
+    return fetch(getRevisionTxtUrl()).then((resp) => {
+      if (!resp.ok) {
+        throw Error(`Error loading revision.txt: ${resp.statusText}`);
```

Comment:
Since the original feature was added, `revision.txt` always exists, even in development.
As such the error handling for it not existing can be removed.